From 40a5d0a4b006780167976b296984eb9790f3df7f Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@tiptoe.de>
Date: Sun, 24 Jun 2018 11:47:05 -0400
Subject: [PATCH] fix TypeReflectionTest for sqlite 3.24

Fixed issue in test suite where SQLite 3.24 added a new reserved word that
conflicted with a usage in TypeReflectionTest.  Pull request courtesy Nils
Philippsen.

Change-Id: I396562cecb5ca774f29e9234845bcc6a399fc5cb
Pull-request: https://github.com/zzzeek/sqlalchemy/pull/452
---
 doc/build/changelog/unreleased_12/pr452.rst | 7 +++++++
 test/dialect/test_sqlite.py                 | 2 +-
 2 files changed, 8 insertions(+), 1 deletion(-)
 create mode 100644 doc/build/changelog/unreleased_12/pr452.rst

diff --git a/doc/build/changelog/unreleased_12/pr452.rst b/doc/build/changelog/unreleased_12/pr452.rst
new file mode 100644
index 000000000..471b203dd
--- /dev/null
+++ b/doc/build/changelog/unreleased_12/pr452.rst
@@ -0,0 +1,7 @@
+.. change::
+    :tags: bug, sqlite
+    :versions: 1.3.0b1
+
+    Fixed issue in test suite where SQLite 3.24 added a new reserved word that
+    conflicted with a usage in TypeReflectionTest.  Pull request courtesy Nils
+    Philippsen.
diff --git a/test/dialect/test_sqlite.py b/test/dialect/test_sqlite.py
index 4c462aed1..d2d563208 100644
--- a/test/dialect/test_sqlite.py
+++ b/test/dialect/test_sqlite.py
@@ -1637,7 +1637,7 @@ def _type_affinity_fixture(self):
             ("BLOBBER", sqltypes.NullType()),
             ("DOUBLE PRECISION", sqltypes.REAL()),
             ("FLOATY", sqltypes.REAL()),
-            ("NOTHING WE KNOW", sqltypes.NUMERIC()),
+            ("SOMETHING UNKNOWN", sqltypes.NUMERIC()),
         ]
 
     def _fixture_as_string(self, fixture):
